Toshi/Source/Toshi/Strings/TCString.h

```diff
@@ -38,8 +38,8 @@ namespace Toshi
 
 	private:
 		char* m_pBuffer = NullString;
-		int m_iExcessLen : 8;
-		int m_iStrLen : 24;
+		int m_iExcessLen : 8 = 0;
```

Comment:
Note: I will change this to an byte because 8 bits int is basically a byte lol

Reply:
maybe also set m_iStrLen to unsigned int since it cannot have a value below 0